src/esw/brushless.py

```diff
@@ -221,16 +247,11 @@ def __init__(self):
         self.drive_bridge_by_name = {}
         using_pi3_hat = rospy.get_param("brushless/using_pi3_hat")
         if using_pi3_hat:
-            # import moteus_pi3hat
-            # transport = moteus_pi3hat.Pi3HatRouter(
-            #     servo_bus_map={
-            #         1: [11],
-            #         2: [12],
-            #         3: [13],
-            #         4: [14],
-            #     },
-            # )
-            pass
+            import moteus_pi3hat
```

Comment:
I take it moteus-pi3hat isnt in the requirements.txt because you cant install it on a non arm computer?

Reply:
Good q. U can only install on pi3. Last year we had a separate requirements txt for Jetson, basestation, and pi. Should we do that still?

Reply:
are there any more device specific requirements other than this?

Reply:
None that I know of

Reply:
then I think we could either leave it as is or we could make a requirements_pi.txt file with just this in it, idrc which

Reply:
we can leave it as is for now, we can just add it to the wiki for now